otx/algorithms/action/adapters/mmaction/models/detectors/fast_rcnn.py

```diff
@@ -69,6 +75,8 @@ def __init__(self, backbone, roi_head, train_cfg, test_cfg, neck=None, pretraine
             pretrained=pretrained,
         )
         self.detector = None
+        self.deply_cfg = None
+        self.detector_cfg = Config({"model": faster_rcnn})
 
     def add_detector(self):
```

Comment:
How about making `add_detector` and `patch_pools` as internal methods and creating a public method calling aforementioned methods?

Reply:
Good opinion. I added a public method patch_for_export and it calls internal methods _add_detector, _patch_pools